modules/utxo-lib/src/networks.ts

```diff
@@ -35,6 +35,21 @@ const coins = {
   DOGE: 'doge',
 } as const;
 
+const testnetCoins = {
```

Comment:
as described in the `coins` const - we don't actually want or need to define coin names here, we already have it in the `statics` package